### ISSUE TITLE
Missing font-awesome icons in edit orders address block

### ISSUE THREAD

**Issue by daphilli224** [lat9/edit_orders#235]
The icons (customer, billing and shipping) are missing on the admin edit orders page address blocks. They are displayed on the orders details page. Should be an easy fix, but I don't know where to fix it. 
Dave
zc158a, php 8.2, edit orders 4.7.0, sba, bootstrap 3.6.3.

**Comment by daphilli224** [comment]
Some more information...I found that font-awesome classes fa-lg, fa-solid, and fa-regular did not show up in a search of admin/includes/css/font-awesome.min.css. On the order details screen, classes fa and fa-2x were used (in addition to fa-user, fa-credit-card, and fa-truck). I deleted fa-solid and fa-regular from admin/includes/modules/edit_orders/eo_edit_action_addresses_display.php in lines 30, 45, 51, 67, and 73, and replaced fa-lg with fa fa-2x on line 31 in admin/includes/modules/edit_orders/eo_common_address_format.php. The icons were then displayed normally in the edit orders page.

But... fa-lg is used in many other places in zc1.5.8a, so I can not explain what's going on.

I hope this provides some clues to a fix.

Thanks for the new update!
